electron/preload/index.ts

```diff
@@ -38,6 +38,10 @@ type Commands = {
         hardfork: `${Hardfork}`;
       };
       walletContractAddress?: string;
+      durableNonce?: {
+        publicKey: string;
+        secretKey: string;
+      };
```

Comment:
we really gotta fix this type soon-ish...

Reply:
Completely agree, maybe we can have a chat about to go about this. Could potentially use Parameters generic type to get parameters of recover function for each coin, and create a recover function in WRW for each coin. Not sure if there is another solution available, but I'm all ears.